scripts/generate_ssl_debug_helpers.py

```diff
@@ -286,34 +283,24 @@ def generate_ssl_debug_helpers(output_directory, mbedtls_root):
         source_code = remove_c_comments(f.read())
 
     definitions = dict()
-    prototypes = dict()
     for start, instance in preprocess_c_source_code(source_code, EnumDefinition):
         if start in definitions:
             continue
         if isinstance(instance, EnumDefinition):
-            definition, prototype = instance.generate_tranlation_function()
+            definition = instance.generate_tranlation_function()
```

Comment:
Nit, preexisting: `generate_tranlation_function()` -> `generate_translation_function()` and in declaration etc.

Reply:
Preexisting, so I'm definitely not fixing it here. This is an urgent fix for the CI, not a general cleanup.

Reply:
Fix at #5348.